Stripe/PublicHeaders/STPRedirectContext.h

```diff
@@ -44,15 +44,37 @@ typedef NS_ENUM(NSUInteger, STPRedirectContextState) {
  @param error An error if one occured. Note that a lack of an error does not 
  mean that the action was completed successfully, the presence of one confirms 
  that it was not. Currently the only possible error the context can know about 
- is if SFSafariViewController fails its initial load (like the user has no 
+ is if SFSafariViewController fails its initial load (e.g. the user has no
  internet connection, or servers are down).
  */
-typedef void (^STPRedirectContextCompletionBlock)(NSString *sourceID, NSString * __nullable clientSecret, NSError * __nullable error);
+typedef void (^STPRedirectContextSourceCompletionBlock)(NSString *sourceID, NSString * __nullable clientSecret, NSError * __nullable error);
 
 /**
- This is a helper class for handling redirect sources.
+ A callback run when the context believes the redirect action has been completed.
```

Comment:
for here and below `A callback that is run when` reads more clearly to me than `A callback run`

Reply:
maybe `A callback that is executed` for consistency with the rest of my comments